session.go

```diff
@@ -372,24 +390,24 @@ func (s *Session) rejectSubscription(id uint64, errorCode uint64, reason string)
 
 func (s *Session) unsubscribe(id uint64) error {
 	return s.ctrlMsgSendQueue.enqueue(context.Background(), &wire.UnsubscribeMessage{
-		SubscribeID: id,
+		RequestID: id,
 	})
 }
 
 func (s *Session) subscriptionDone(id, code, count uint64, reason string) error {
 	lt, ok := s.localTracks.delete(id)
 	if !ok {
-		return errUnknownSubscribeID
+		return errUnknownRequestID
 	}
 	return s.ctrlMsgSendQueue.enqueue(context.Background(), &wire.SubscribeDoneMessage{
-		SubscribeID:  lt.subscribeID,
+		RequestID:    lt.requestID,
 		StatusCode:   code,
 		StreamCount:  count,
 		ReasonPhrase: reason,
 	})
 }
 
-// Fetch fetches track in namespace from the peer using id as the subscribe ID.
+// Fetch fetches track in namespace from the peer using id as the request ID.
```

Comment:
Again, this is generating a new requestID, not an id parameter. I think that the requestID value should be returned, so that one can refer to it later.

Reply:
Also removed the part from the comment here. In the past, there was a parameter and when I removed it, I forgot to update the comment.